src/util/util_log.cpp

```diff
@@ -56,10 +56,17 @@ namespace dxvk::log {
         if (skipAllLogging)
             return;
 
-        print(message);
+        auto logMessage = str::format(
+            std::setfill('0'), std::setw(4), std::hex, ::GetCurrentProcessId(), ":",
+            std::setfill('0'), std::setw(4), std::hex, ::GetCurrentThreadId(), ":",
+            "info:",
+            "dxvk-nvapi:",
+            message);
```

Comment:
Pretty weird that we do the formatting of the message here without the trailing newline only to add it separately below and inside `print`. I assume this is because we're using `print` somewhere else directly?

Reply:
I guess we wanted to prevent output flushing when writing to wine, https://stackoverflow.com/a/213977